mathesar_ui/src/i18n/i18n-load.ts

```diff
@@ -30,3 +30,11 @@ export async function loadLocaleAsync(locale: Locales): Promise<void> {
   updateTranslationsDictionary(locale, await importLocaleAsync(locale));
   loadFormatters(locale);
 }
+
+export function loadTranslationsIntoMemory(
```

Comment:
Can this function be named `loadTranslations` instead? The name `loadTranslationsIntoMemory` throws me off a bit because it's in the same file as other functions which seem to be also loading things _into memory_ but are not named in a similar manner.

Reply:
1738aca